bubble-wrap.yaml

```diff
@@ -959,7 +958,7 @@ styles:
 
 scene:
     background:
-        color: global.scene1
+        color: [0.870,0.870,0.870]
```

Comment:
Could be moved into a global, but wasn't before anyway (see old earth layer below).